js/app.js

```diff
@@ -47,7 +47,7 @@ drinkIngredients.prototype = {
             order = "<h2>Choose some ingredients you landlubber!</h2>";
             $('div.results').append(order);
         } else {
-            $('div.results').append('<hr><h2>Here\'s your Drink Matey in your ' + container + '</h2><ul>' + order + '</ul>');
+            $('div.results').append('<hr><h2>Here\'s your drink matey in your ' + container + '</h2><ul>' + order + '</ul>');
```

Comment:
Thanks for the feedback!

On Sat, Aug 13, 2016 at 1:43 PM, Alex Weldetsadik <notifications@github.com>
wrote:

> In js/app.js
> <https://github.com/danj707/pirate-bartender/pull/1#discussion_r74690569>:
>
> > @@ -47,7 +47,7 @@ drinkIngredients.prototype = {
> >              order = "<h2>Choose some ingredients you landlubber!</h2>";
> >              $('div.results').append(order);
> >          } else {
> > -            $('div.results').append('<hr><h2>Here\'s your Drink Matey in your ' + container + '</h2><ul>' + order + '</ul>');
> > +            $('div.results').append('<hr><h2>Here\'s your drink matey in your ' + container + '</h2><ul>' + order + '</ul>');
>
> Have you considered using an <li> </li> for the order?
>
> $('div.results').append('<hr><h2>Here\'s your drink matey in your ' + container + '</h2><ul><li>' + order + '</li></ul>');
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/danj707/pirate-bartender/pull/1/files/aa89eccb54f5ad144e3f0da86e537526704e9a3a..7bc254c95d9f69402c1769cc9f84e39539195a3d#r74690569>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARa88u6NdRDNQczOOsfAPEElbW7eRrZBks5qfhBZgaJpZM4JjOF->
> .
>